### PR TITLE
Update MsgObs for Cors PR

### PR DESCRIPTION
**Pull request by denniszollo**
requires merge of libsbp PR https://github.com/swift-nav/libsbp/pull/354

### CODE REVIEW

File: src/base_obs.c [hunk 1]
```diff
@@ -351,7 +351,7 @@ static void obs_callback(u16 sender_id, u8 len, u8 msg[], void* context)
 static void deprecated_callback(u16 sender_id, u8 len, u8 msg[], void* context)
 {
   (void) context; (void) len; (void) msg; (void) sender_id;
-  log_error("Receiving an old deprecated observation message.");
+  log_error("Received a deprecated observation message. Verify firmware version on local and remote Piksi.");
```
Comment (jacobmcnamee) [inline src/base_obs.c]: This should be a warning IMO

Comment (jacobmcnamee) [inline src/base_obs.c]: Will the longer string show up properly in piksi console?

Comment (denniszollo) [inline src/base_obs.c]: agree

**Comment by ljbade** [comment]
@denniszollo Seems like this PR now has other changes in it too from other branches?

**Comment by denniszollo** [comment]
WIP progress -

**Pull request closed without merging**